components/tikv_util/src/logger/file_log.rs

```diff
@@ -32,85 +34,94 @@ fn open_log_file(path: impl AsRef<Path>) -> io::Result<File> {
     OpenOptions::new().append(true).create(true).open(path)
 }
 
-/// This FileLogger rotates logs according to a time span.
+fn rename_by_timestamp(path: impl AsRef<Path>) -> io::Result<PathBuf> {
```

Comment:
This function performs the rotate / rename by current time (instead of by timestamp), better to consider a new function name.